src/Functions/multiSearchFirstPositionCaseInsensitiveUTF8.cpp

```diff
@@ -21,7 +21,31 @@ using FunctionMultiSearchFirstPositionCaseInsensitiveUTF8 = FunctionsMultiString
 
 REGISTER_FUNCTION(MultiSearchFirstPositionCaseInsensitiveUTF8)
 {
-    factory.registerFunction<FunctionMultiSearchFirstPositionCaseInsensitiveUTF8>();
+    FunctionDocumentation::Description description = R"(
+Like [multiSearchFirstPosition](#multiSearchFirstPosition) but assumes `haystack` and `needle` to be UTF-8 strings and ignores case.
+    )";
+    FunctionDocumentation::Syntax syntax = "multiSearchFirstPositionCaseInsensitiveUTF8(haystack, [needle1, needle2, ..., needleN])";
+    FunctionDocumentation::Arguments arguments = {
+        {"haystack", "UTF-8 string in which the search is performed.", {"String"}},
+        {"needle", "Array of UTF-8 substrings to be searched.", {"Array(String)"}}
+    };
+    FunctionDocumentation::ReturnedValue returned_value = {"Leftmost offset in a `haystack` string which matches any of multiple `needle` strings, ignoring case. Returns `0`, if there was no match.", {"UInt64"}};
+    FunctionDocumentation::Examples examples = {
+    {
+        "Case insensitive UTF-8 first position",
```

Comment:
\+ `Find the leftmost offset in UTF-8 string HELLO WORLD which matches any of the given needles.`